tests/stdlib/tos.nim

```diff
@@ -261,6 +261,21 @@ block fileOperations:
 
     removeDir(dname)
 
+
+    block:
+      const dname = buildDir/"D20210116T1406234"
+      const symlinkName = dname/"D20210101T191320_BROKEN_SYMLINK"
+      const symlinkSrc = buildDir.parentDir/"D20210101T1920_nonexistant"
+
+      createDir(dname)
+
+      let f = open(symlinkSrc, fmWrite)
+      f.close()
```

Comment:
* I don't understand this; you're creating a file `"D20210101T1920_nonexistant"` (nonexistant should indicate it doesn't exist)
* generated files should go under buildDir (which is gitignored), but `symlinkSrc` is not

simply use:
```nim
const symlinkSrc = "D20210101T1920_nonexistant"
createSymlink(symlinkSrc, symlinkName)
doAssert expandSymlink(symlinkName) == symlinkSrc
```